mappings/net/minecraft/world/biome/source/util/MultiNoiseUtil.mapping

```diff
@@ -1,46 +1,55 @@
-		ARG 0 parameter
-	METHOD method_37621 createParameterConfig (FF)Lnet/minecraft/class_6452$class_6454;
-		ARG 0 parameter1
-		ARG 1 parameter2
+	FIELD field_34168 PARAMETER_NUMBER I
+	METHOD method_37620 createParameterRange (F)Lnet/minecraft/class_6452$class_6454;
+		ARG 0 point
+	METHOD method_37621 createParameterRange (FF)Lnet/minecraft/class_6452$class_6454;
+		ARG 0 min
+		ARG 1 max
 	METHOD method_37622 createNoiseValuePoint (FFFFFF)Lnet/minecraft/class_6452$class_6460;
 		ARG 0 temperatureNoise
 		ARG 1 humidityNoise
 		ARG 2 continentalnessNoise
 		ARG 3 erosionNoise
+		ARG 4 depth
 		ARG 5 weirdnessNoise
-	METHOD method_37623 createMixedNoisePoint (FFFFFFF)Lnet/minecraft/class_6452$class_4762;
+	METHOD method_37623 createNoiseHypercube (FFFFFFF)Lnet/minecraft/class_6452$class_4762;
 		ARG 0 temperature
 		ARG 1 humidity
 		ARG 2 continentalness
 		ARG 3 erosion
 		ARG 4 depth
 		ARG 5 weirdness
 		ARG 6 offset
-	METHOD method_37625 createMixedNoisePoint (Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;F)Lnet/minecraft/class_6452$class_4762;
+	METHOD method_37624 combineParameterRange (Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;)Lnet/minecraft/class_6452$class_6454;
+		COMMENT Creates a new {@link MultiNoiseUtil.ParameterRange} that combines the parameters.
+		COMMENT
+		COMMENT @return the created parameter range.
+		ARG 0 min
+			COMMENT this will be used for the created range's minimum value.
+		ARG 1 max
+			COMMENT this will be used for the created range's maximum value.
+	METHOD method_37625 createNoiseHypercube (Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;F)Lnet/minecraft/class_6452$class_4762;
 		ARG 0 temperature
 		ARG 1 humidity
 		ARG 2 continentalness
 		ARG 3 erosion
 		ARG 4 depth
 		ARG 5 weirdness
 		ARG 6 offset
-	CLASS class_4762 MixedNoisePoint
-		COMMENT Represents a point in a multi-dimensional cartesian plane. Mixed-noise
-		COMMENT biome generator picks the closest noise point from its selected point
-		COMMENT and choose the biome associated to that closest point. Another factor,
-		COMMENT rarity potential, favors larger differences in values instead, contrary
-		COMMENT to other point values.
+	CLASS class_4762 NoiseHypercube
+		COMMENT Represents a hypercube in a multi-dimensional cartesian plane. The multi-noise
+		COMMENT biome source picks the closest noise hpercube from its selected point
+		COMMENT and chooses the biome associated to it. Another factor, rarity potential,
+		COMMENT favors larger differences in values instead, contrary to other range values.
```

Comment:
Hmm, don't think this factor exists any more. I think they now use an offset, where when they sample, the offset is always 0, and the biome target cube points have a non-0 offset

Reply:
I think that's how `offset` worked before as well, though.

Reply:
Can just delete this "Another factor, ..." sentence then

Reply:
I think "another factor" doesn't mean the name of it here, but that it's another thing that contributes to the biome layout.

Reply:
```suggestion
```